### PR TITLE
Stop using hard coded ga keys

### PR DESCRIPTION
**Pull request by rawOrlando**
Some clean up I forgot from the ga work.

### CODE REVIEW

File: app/views/analytics/_tag_manager_body.html.erb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <% if Rails.env.production? %>
+<% file_Path = File.open(Rails.root.join('config', 'ga.yml').to_s) %>
+
+<% tag_manager_key = YAML.safe_load(File.open(Rails.root.join('config', 'ga.yml').to_s)).deep_symbolize_keys![Rails.env.to_sym][:tag_manager_key] %>
```
Comment (rawOrlando) [inline app/views/analytics/_tag_manager_body.html.erb:4]: @mkorcy Is there a better way to get this data here? Does it involve config?

Comment (mkorcy) [reply app/views/analytics/_tag_manager_body.html.erb:4]: You could move this to a helper or application controller and do something like:
```

   def tag_manager_key
    return nil unless Rails.env.production?
    ga_config_path = Rails.root.join('config', 'ga.yml')
    ga_config = YAML.safe_load(File.read(ga_config_path)).deep_symbolize_keys
    ga_config[Rails.env.to_sym][:tag_manager_key]
  end

```

and then in the view do :

`<% key = tag_manager_key %>`

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -35,4 +37,20 @@ def default_url_options
...
+    ga_config[Rails.env.to_sym][:api_key]
+  end
+
+  def ga_tag_manager_key
+    ga_config_path = Rails.root.join('config', 'ga.yml')
+    ga_config = YAML.safe_load(File.read(ga_config_path)).deep_symbolize_keys
+    return nil unless ga_config.key?(Rails.env.to_sym)
+    return nil unless ga_config[Rails.env.to_sym].key?(:tag_manager_key)
+    ga_config[Rails.env.to_sym][:tag_manager_key]
+  end
```
Comment (mkorcy) [inline app/controllers/application_controller.rb:55]: How about a refactor here like:

```
class ApplicationController < ActionController::Base
  helper_method :ga_api_key, :ga_tag_manager_key

  ...

  def ga_tag_manager_key
    fetch_ga_config(:tag_manager_key)
  end

  private

  def fetch_ga_config(key)
    config = load_ga_config
    return nil unless config&.key?(key)
    ga_config[key]
  end

  def load_ga_config
    ga_config_path = Rails.root.join('config', 'ga.yml')
    configs = YAML.safe_load(File.read(ga_config_path)).deep_symbolize_keys
    configs[Rails.env.to_sym]
  end
end
```

**Review by mkorcy — CHANGES_REQUESTED** [review]
i proposed one refactor the rest looks good.

**Pull request merged**